pgx-pg-config/src/lib.rs

```diff
@@ -451,9 +451,22 @@ pub fn createdb(
         .arg(dbname)
         .stdout(Stdio::piped())
         .stderr(Stdio::piped());
+
     let command_str = format!("{:?}", command);
 
-    let output = command.output()?;
+    let child = command.spawn().wrap_err_with(|| {
+        format!(
+            "Failed to spawn process for creating database using command: '{}': ",
+            command_str
```

Comment:
These format strings can have problems if you rely on them indexing like that, so I generally recommend avoiding that. Also tends to eat source-level vertical space.
```suggestion
            "Failed to spawn process for creating database using command: '{command_str}': "
```

Reply:
Ah right, I keep forgetting you can do that with `format!()`.